app/scripts/icons.js

```diff
@@ -369,3 +371,18 @@ export const svgArrowheadDomainsIcon = parser.parseFromString(
   arrowHeadDomainsStr,
   'text/xml',
 ).documentElement;
+
+export function SearchIcon({ theStyle }) {
```

Comment:
How about calling the property `className` so it's clearer what this is support to be?

Reply:
I'm afraid that that'll interfere with other functionality. `styleName` is how we pass in styles in other components.